### PR TITLE
Add replace_with_parallel_cross_entropy flag

### PR DESCRIPTION
**Pull request by waliwali777**
### PR types

Others
### PR changes

Others
### Description

为 `replace_with_parallel_cross_entropy pass` 添加开关，可以在脚本进行开关控制
开 pass： `--tensor_parallel_config "replace_with_parallel_cross_entropy"`
关 pass： `--tensor_parallel_config ""`

### CODE REVIEW

File: llm/auto_parallel/llama/run_pretrain_auto.sh [hunk 1]
```diff
@@ -74,6 +74,7 @@ python -u  -m paddle.distributed.launch \
     --do_eval \
     --device "gpu" \
     --data_impl "mmap" \
+    --replace_with_parallel_cross_entropy 0 \
```
Comment (liym27) [inline llm/auto_parallel/llama/run_pretrain_auto.sh]: Would it be better to place `replace_with_parallel_cross_entropy`  in `tensor_parallel_config` configuration?

Comment (waliwali777) [reply llm/auto_parallel/llama/run_pretrain_auto.sh]: Done
Move `replacew_ith_marallel_cross_detropy ` to `tensor_marallel_config ` for configuration.

File: scripts/distribute/ci_case_auto.sh [hunk 2]
```diff
@@ -839,6 +840,103 @@ function llama_pir_auto_fuse_ffn_attention_qkv_MP2() {
     echo "=========== $FUNCNAME run  end ==========="
 }
 
+function llama_pir_auto_replace_with_parallel_cross_entropy_MP2() {
```
Comment (liym27) [inline scripts/distribute/ci_case_auto.sh]: 这样的写法会增加2个测试。建议直接在已有的测试上，加上 parallel cross entropy 参数验证，只需增加一个测试。

Comment (waliwali777) [reply scripts/distribute/ci_case_auto.sh]: done

**Review by liym27 — APPROVED** [review]
LGTM

**Pull request merged**